pom.xml

```diff
@@ -88,7 +88,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-validator</artifactId>
-            <version>5.2.2.Final</version>
+            <version>5.3.0.Alpha1</version>
```

Comment:
@gmandnepr do you think it is fine to use alpha version? Maybe we should use latest stable one?